test_runner/build.gradle.kts

```diff
@@ -60,11 +60,21 @@ java {
 }
 
 publishing {
+    repositories {
+        maven {
+            name = "GitHubPackages"
+            url = uri("https://maven.pkg.github.com/Flank/flank")
+            credentials {
+                username = System.getenv("GITHUB_ACTOR") ?: properties["GITHUB_ACTOR"].toString()
+                password = System.getenv("GITHUB_TOKEN") ?: properties["GITHUB_TOKEN"].toString()
+            }
+        }
+    }
     publications {
         create<MavenPublication>("mavenJava") {
             groupId = "com.github.flank"
             artifactId = artifactID
-            version = System.getenv("MVN_VERSION")
+            version = System.getenv("MVN_VERSION") ?: "local_snapshot"
```

Comment:
If my understanding is correct there is no `MVN_VERSION` if we are publishing snapshot (after the merge to master). How about naming it just `snapshot`? It's not a change request, I'd like just to fully understand it :)

Reply:
MVN_VERSION is set by CI, `local_snapshot` is used for local builds. Its value is:

- `flank_snapshot` for snapshot build
- `xx.xx.xx` for normal builds

I used `local_snapshot` cause it is also used for printing local version

Reply:
Got it, thanks for the explanation! 👍